openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -322,6 +322,42 @@ This specification relies on certain prerequisites, such as browser or operating
 
 Wallet implementations using the key attestation format specified in Annex D of [@!OIDF.OID4VCI] might need to utilize a transformation (backend) service to create such attestations based on data as provided in other formats by the respective platform or secure key management module. The dependency on such a service might impact the availability of the wallet app as well as the performance of the issuance process. This could be mitigated by creating keys and obtaining the respective key attestations in advance.
 
+## Ecosystem Implementation Considerations
+
+This document leaves extensible a number of different options to allow for broad compatible with ecosystems that have conflicting needs. These include:
+
+- Credential format support requirements across issuance and presentation
+- Signed Issuer Metadata.
+- Credential Offer invocation mechanisms.
+- Key attestation formats.
+- X509 Verifier Authentication certificate profiles.
```

Comment:
Not just verifier authentication, but also token status list verification, credential verification, wallet attestation, and probably more

```suggestion
- X509 certificate profiles.
```